### ISSUE TITLE
SOLVED [Bug] Apps in 'Auto-Launch Folder' do not start upon starting vrchat

### ISSUE THREAD

**Issue by BlackWolfWoof** [vrcx-team/VRCX#838]
**ISSUE SOLVED**
## TLDR; VRCX only starts files with the `lnk` extention. If you use a `url` file like the one steam generates, make sure to go to the source folder and create a shortcut for it. Same goes for any other scripts that you want to run upon starting vrchat.

**Describe the bug**
When using the App Launcher Auto-Launch Folder and turning on the auto launching of apps when starting VRChat, VRCX does not auto start any of the applications in the folder.

**To Reproduce**
1. Put a program / link to a program of your choice into the autostart folder which can be found under `%USERPROFILE%\AppData\Roaming\VRCX\startup` or by clicking "Auto-Launch Folder" in your settings menu.
2. Toggle on "Enable"
3. Start VRChat

**Expected behavior**
VRCX should start the applications listed in the folder upon starting vrchat.

**Screenshots**
N/A

**What version you are running**
Version [e.g. 2024.06.12] stable built

**Comment by Myrkie** [comment]
VRCX will only auto start app shortcuts not programs directly https://github.com/vrcx-team/VRCX/issues/647#issuecomment-1719721933

**Comment by BlackWolfWoof** [comment]
> VRCX will only auto start app shortcuts not programs directly [#647 (comment)](https://github.com/vrcx-team/VRCX/issues/647#issuecomment-1719721933)

Does it need to be an `lnk`? Because i have a ps1 and url in there. 
![image](https://blackwolfwoof.com/s/ins7YYmjYgRKGQa/preview)

**Comment by BlackWolfWoof** [comment]
Thanks for the quick reply @Myrkie 🖤
